
string robot_name
std_msgs/Float64 people_seen